docsrc/content/api.fsx

```diff
@@ -105,6 +105,28 @@ let nonString = Not AnyString
 AnyNumber
 
 (**
+#### Validate a number with specific criteria
+```fsharp
+NumberThat of NumberAttributes list
+```
+*)
+(**
+Where `NumberAttributes` can be any valid combination of
+
+  - `IsPositive` - ensure that the number is positive (greater than 0).
+
+  - `IsNegative` - ensure that the number is negative (less than 0).
+
+  - `IsNonNegative` - ensure that the number is 0 or greater.
+
+  - `IsGreaterThan` - ensure that the number is greater than a given number.
+
+  - `IsLessThan` - ensure that the number is less than a given number.
+
+example: schema to match that a number is positive, greater than 10 and less than 50
+*)
+let numberSchema = NumberThat [IsPositive; IsGreaterThan 10M; IsLessThan 50M]
```

Comment:
I dig this. It's worth noting though that the `IsPositive` is redundant as the `>` and `<` ranges are a subset of the positive numbers. Nothing major but it might be confusing for folks.